fairmq/shmem/Socket.cxx

```diff
@@ -5,26 +5,46 @@
  *              GNU Lesser General Public Licence (LGPL) version 3,             *
  *                  copied verbatim in the file "LICENSE"                       *
  ********************************************************************************/
+
 #include "Common.h"
+#include "Socket.h"
+#include "Message.h"
+#include "UnmanagedRegion.h"
+#include "TransportFactory.h"
 
-#include "FairMQSocketSHM.h"
-#include "FairMQMessageSHM.h"
-#include "FairMQUnmanagedRegionSHM.h"
-#include "FairMQLogger.h"
+#include <FairMQLogger.h>
 #include <fairmq/Tools.h>
 
 #include <zmq.h>
 
 #include <stdexcept>
 
 using namespace std;
-using namespace fair::mq::shmem;
-using namespace fair::mq;
 
-atomic<bool> FairMQSocketSHM::fInterrupted(false);
+namespace fair
+{
+namespace mq
+{
+namespace shmem
+{
+
+atomic<bool> Socket::fInterrupted(false);
+
+struct ZMsg
+{
+    ZMsg()            { int rc __attribute__((unused)) = zmq_msg_init(&fMsg);            assert(rc == 0); }
+    ZMsg(size_t size) { int rc __attribute__((unused)) = zmq_msg_init_size(&fMsg, size); assert(rc == 0); }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Struct 'ZMsg' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/app/FairRootGroup/FairMQ/pullRequest?prid=4713729)